packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point.tsx

```diff
@@ -20,10 +21,16 @@ type Props = {
 const hitboxSizePx = 48;
 
 export const StyledMovablePoint = (props: Props) => {
-    const {range, snapStep, markings} = useGraphConfig();
+    const {range, snapStep, markings, showTooltips} = useGraphConfig();
     const hitboxRef = useRef<SVGCircleElement>(null);
     const {point, onMove, color = WBColor.blue} = props;
```

Comment:
Note that as far as I know, there are no plans to allow content authors to adjust the color of any of the movables. That is something that is strictly reserved for Locked Features. 

So if it simplifies things, I think we can remove the ability to change the color of this StyledMovablePoint.